grr/server/grr_response_server/gui/api_plugins/metadata.py

```diff
@@ -37,3 +37,25 @@ def Handle(
     result.revision = version_dict["revision"]
     result.release = version_dict["release"]
     return result
+
+
+class ApiGetOpenApiDescriptionResult(rdf_structs.RDFProtoStruct):
+  protobuf = metadata_pb2.ApiGetOpenApiDescriptionResult
+
+
+class ApiGetOpenApiDescriptionHandler(api_call_handler_base.ApiCallHandler):
+  """Renders a description of the API using the OpenAPI specification."""
+
+  args_type = None
+  result_type = ApiGetOpenApiDescriptionResult
+
+  def Handle(
+      self,
+      args: None,
+      token: Optional[access_control.ACLToken] = None,
+  ) -> ApiGetOpenApiDescriptionResult:
```

Comment:
Same for public methods.

Reply:
Added them. Thank you and please let me know if they're not ok.